ddln_imaginator.proto

```diff
@@ -4,15 +4,26 @@ package ddln;
 
 import "ddln_common.proto";
 
+// Imaginator interface definitions
+//
+// This file contains interface definitions for Daedalean's cloud image generator: Imaginator.
+// Imaginator is a cloud rendering service which takes image requests and returns generated images.
+// Typically an image request consists of camera intrinsics, extrinsics, environmental conditions, and so on.
+// Daedalean provides a range of scenery using different cloud endpoints.
+// Contact sim@daedalean.ai for more information and usage.
+//
+// Note 1: Interface units typically are assumed in metric system, e.g. SI units, unless indicated otherwise.
+// Note 2: World coordinates are expressed in local East North Up (ENU) coordinate frame
+// Note 3: Other coordinate frames are defined in ddln_common.proto
+//
+
 message CameraIntrinsics {
     required float verticalFieldOfView_deg = 1;
 }
 
 message SceneObject {
     optional string type = 1;
-
-    // pose in ENU coordinates
-    optional Pose pose = 2;
+    optional Pose pose_WA = 2; // pose of object A in world frame W (see ddln_common.proto)
```

Comment:
move the comment above the property definition, don't use same-line comments in this file please